### PR TITLE
snos release patches

### PR DESCRIPTION
**Pull request by apoorvsadana**

### CODE REVIEW

File: crates/orchestrator/src/workers/snos.rs [hunk 1]
```diff
@@ -17,53 +17,32 @@ impl Worker for SnosWorker {
...
 
-        // if all blocks are processed
-        if block_diff == 0 {
-            tracing::info!("All blocks are already processed");
-            return Ok(());
-        }
-
-        for x in latest_block_processed + 1..latest_block_number + 1 {
-            create_job(JobType::SnosRun, x.to_string(), HashMap::new(), config.clone()).await?;
+        for x in block_start..latest_block_number + 1 {
```
Comment (Mohiiit) [inline crates/orchestrator/src/workers/snos.rs:44]: change variable for better readability

File: crates/orchestrator/src/workers/update_state.rs [hunk 2]
```diff
@@ -1,136 +1,153 @@
...
+                    .unwrap()
+                    .split(',')
+                    .filter_map(|s| s.parse().ok())
+                    .collect();
+
+                // ideally it's already sorted, but just to be safe
+                blocks_processed_in_last_job.sort();
+
+                let last_block_processed_in_last_job =
+                    blocks_processed_in_last_job[blocks_processed_in_last_job.len() - 1];
```
Comment (Mohiiit) [inline crates/orchestrator/src/workers/update_state.rs:43]: can there be case when we have `JOB_METADATA_STATE_UPDATE_BLOCKS_TO_SETTLE_KEY` empty?

**Review by ocdbytes — APPROVED** [review]
lgtm !!

**Review by byteZorvin — APPROVED** [review]
lgtm

**Pull request merged**